Crust/Utilities/ArrayAdaptor.swift

```diff
@@ -22,11 +22,11 @@ open class ArrayMapping<SubType: Equatable, SubAdaptor: Adaptor, SubMapping: Arr
     
     public var primaryKeys: [String : Keypath]? { return nil }
     open var keyPath: Keypath { return "" }
-    open var options: MappingOptions { return MappingOptions.None }
+    open var options: CollectionUpdatePolicy<SubMapping.SequenceKind> { return (.append, true) }
```

Comment:
If it's only an update policy instead of a bit mask, I might rename the propery to reflect that.